README.md

```diff
@@ -104,8 +114,22 @@ You can also build individual targets using below commands
 $ make appdynamicsiotsdk
 $ make sample
 $ make tests
+$ sudo make run-code-coverage
+```
+
+You can run tests from the `build` folder by using commands below
+```sh
+$ make ./run-tests
+```
+
+You can run tests from the `build` folder by using commands below
```

Comment:
Text needs to be updated to reflect code coverage 
- e.g., You can run tests with code coverage

Reply:
Done